### PR TITLE
Fix #470 onsite2.researchintel.com

### LINKED ISSUE #470: onsite2.researchintel.com
### Steps to reproduce

I got a customer survey from Intel related to a ticket.
When I want start the Survey, the redirected destination URL is blocked by AdGuard DNS filter.

So... not sure if this related URL is used for Spyware, but to fill out a survey is my own decision and shouldn't be blocked then.

### Expected behavior

<details><summary>Screenshot:</summary>

intel1

intel2

</details>

### Actual behavior

<details><summary>Screenshot:</summary>

intel 3

</details>

### Your environment

|   Description  |     Value    |
| -------------- | ------------ |
| DNS server in use:| 1.1.1.1
| How did you setup DNS configuration:| AdGuard Home Docker
| Device model:| PC Master Race
| Operating system and version:| Windows 10 2004 Build 19041.546

### PR DESCRIPTION
**Pull request by Alex-302**
https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/470

### CODE REVIEW

**Comment by AdamWr** [comment]
Is there any reason why it was added to `exceptions.txt`, I thought that `exclusions.txt` is preferred to fix these kind of issues?
Just asking :)

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
> I thought that `exclusions.txt` is preferred to fix these kind of issues?
> Just asking :)

I have the same question :)

**Comment by Alex-302** [comment]
moved

File: Filters/exclusions.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/470
+||researchintel.com^|
```
Comment (AdamWr) [inline Filters/exclusions.txt:2]: Shouldn't it be without `|` at the end?

Comment (Alex-302) [reply Filters/exclusions.txt]: Fixed.

**Review by AdamWr — APPROVED** [review]

**Pull request merged**